### PR TITLE
Pt 165558593 fix ga meta fees

### PR DESCRIPTION
**Pull request by gorillainduction**
https://www.pivotaltracker.com/story/show/165558593

### CODE REVIEW

File: apps/aega/src/aega_meta_tx.erl [hunk 1]
```diff
@@ -210,6 +212,23 @@ set_meta_result(Err = {error, _}, Tx, Trees, Env) ->
     {ok, Trees1, _Env} = aeprimop:eval(SetInstructions, Trees, Env),
     Trees1.
 
+-spec call_id(tx(), aec_trees:trees()) -> aect_call:id().
```
Comment (hanssv) [inline apps/aega/src/aega_meta_tx.erl]: This could be used in #2357 perhaps?

Comment (gorillainduction) [reply apps/aega/src/aega_meta_tx.erl]: I was wondering who should rebase, but I agree with you ;-)

Comment (hanssv) [reply apps/aega/src/aega_meta_tx.erl]: You and Thomas can fight that out I think ;-) I'll rebase on the end result regardless...

Comment (gorillainduction) [reply apps/aega/src/aega_meta_tx.erl]: @ThomasArts ^^

Comment (hanssv) [reply apps/aega/src/aega_meta_tx.erl]: One alternative is that we merge both, and then I'll resolve it in my upcoming PR...

Comment (hanssv) [reply apps/aega/src/aega_meta_tx.erl]: Looking at the two functions again (with a fresh cup of coffee) I think they are short enough and different enough not to require the extra effort...

**Pull request merged**